### PR TITLE
spec: drop "any" suffix for user mode only motr rpm name

### PR DESCRIPTION
**Pull request by madhavemuri**
Currently "any" suffix in usermode only rpm name represents that
it can work with any kernel version. Use of "any" suffix is not a
standard practice for rpm name, without it too same can be represented,
so removing it.

Also enabled powertools repo for quick package installation option.
Added python3-devel as dependency in cortx-motr.spec.
Fixed echo statement in build prepare scripts for MGW deployment.

Signed-off-by: Madhavrao Vemuri <madhav.vemuri@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: scripts/build-prep-1node-cortx-mgw.sh [hunk 1]
```diff
@@ -88,10 +88,11 @@ echo '[client.rgw.8000]'
...
-echo 'Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs'
+echo "Current directory: $PWD"
+echo "Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs"
 echo 'check:     hctl status'
 echo 'm0client should be in a started state'
 echo 'To stop the cluster run: ../src/stop.sh'
 echo 'Then:  hctl shutdown'
 echo 'If local directory is used then after reboot of the node,'
-echo 'run: mount "/dev/$available_device" $CORTX_WDIR'
+echo "run: mount "/dev/$available_device" $CORTX_WDIR"
```
Comment (andriytk) [inline scripts/build-prep-1node-cortx-mgw.sh:98]: There are multiple `"` quotes here - how is this working?

File: scripts/build-prep-1node-cortx-mgw.sh [hunk 2]
```diff
@@ -88,10 +88,11 @@ echo '[client.rgw.8000]'
 echo '      ...'
 echo '      motr my endpoint  = inet:tcp:10.0.0.1@5001'
 echo '      motr my fid       = 0x7200000000000001:0x29'
-echo 'Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs'
+echo "Current directory: $PWD"
+echo "Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs"
```
Comment (andriytk) [inline scripts/build-prep-1node-cortx-mgw.sh:92]: Do you have `msg` variable to show here?

File: scripts/build-prep-1node-cortx-mgw.sh [hunk 3]
```diff
@@ -71,13 +71,13 @@ echo "========================================="
...
-echo 'Do not use "/dev/$available_device" in cluster yaml'
+echo "Do not use "/dev/$available_device" in cluster yaml"
 echo 'Also do not use the devices with filesystem on them'
 echo 'Now you are ready to start the singlenode Motr cluster!'
-echo 'To start, run: hctl bootstrap --mkfs singlenode.yaml'
+echo "To start, run: hctl bootstrap --mkfs  $CORTX_WDIR/singlenode.yaml"
 echo 'check:     hctl status'
 echo 'To connect to Motr cluster, add the following configuration'
-echo 'parameters from `hctl status` to build/ceph.conf:'
+echo "parameters from `hctl status` to $CORTX_WDIR/cortx-rgw/build/ceph.conf:'
```
Comment (andriytk) [inline scripts/build-prep-1node-cortx-mgw.sh:80]: Check opening and closing quotes here.

File: cortx-motr.spec.in [hunk 4]
```diff
@@ -127,7 +127,9 @@ BuildRequires:  systemd-devel
 %if %{rhel} < 8
 BuildRequires:  python-devel
 %else
+# @todo Deprecate python2
 BuildRequires:  python2-devel
+BuildRequires:  python3-devel
```
Comment (andriytk) [inline cortx-motr.spec.in:132]: Don't we need python2 for spiel anymore?

File: scripts/build-prep-1node-cortx-mgw.sh (hunk 2 above)
Comment (madhavemuri) [reply scripts/build-prep-1node-cortx-mgw.sh:92]: yes.

File: cortx-motr.spec.in (hunk 4 above)
Comment (madhavemuri) [reply cortx-motr.spec.in:132]: Keeping python2-devel, until spiel is ported to python3, https://github.com/Seagate/cortx-motr/pull/1375

File: scripts/build-prep-1node-cortx-mgw.sh (hunk 2 above)
Comment (madhavemuri) [reply scripts/build-prep-1node-cortx-mgw.sh:92]: yes, in this script mgs is set to --without-dashboard" by default

Comment (andriytk) [reply scripts/build-prep-1node-cortx-mgw.sh:92]: ok

**Comment by madhavemuri** [comment]
Changes related to spec are working as expected,
with user-mode only,
```Wrote: /root/rpmbuild/SRPMS/cortx-motr-2.0.0-1_git6ae1b6c5.el8.src.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-2.0.0-1_git6ae1b6c5.el8.x86_64.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-devel-2.0.0-1_git6ae1b6c5.el8.x86_64.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-debugsource-2.0.0-1_git6ae1b6c5.el8.x86_64.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-debuginfo-2.0.0-1_git6ae1b6c5.el8.x86_64.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-devel-debuginfo-2.0.0-1_git6ae1b6c5.el8.x86_64.rpm```

with both kernel and user mode support,

```Wrote: /root/rpmbuild/SRPMS/cortx-motr-2.0.0-1_git6ae1b6c5_4.18.0_305.3.1.el8_4.el8.src.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-2.0.0-1_git6ae1b6c5_4.18.0_305.3.1.el8_4.el8.x86_64.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-devel-2.0.0-1_git6ae1b6c5_4.18.0_305.3.1.el8_4.el8.x86_64.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-debugsource-2.0.0-1_git6ae1b6c5_4.18.0_305.3.1.el8_4.el8.x86_64.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-debuginfo-2.0.0-1_git6ae1b6c5_4.18.0_305.3.1.el8_4.el8.x86_64.rpm
Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-devel-debuginfo-2.0.0-1_git6ae1b6c5_4.18.0_305.3.1.el8_4.el8.x86_64.rpm```

File: scripts/build-prep-1node-cortx-mgw.sh (hunk 1 above)
Comment (madhavemuri) [reply scripts/build-prep-1node-cortx-mgw.sh:98]: Checked it, it works fine.

Comment (madhavemuri) [reply scripts/build-prep-1node-cortx-mgw.sh:98]: run: mount /dev/sdc /root/cortx

File: scripts/build-prep-1node-cortx-mgw.sh (hunk 3 above)
Comment (madhavemuri) [reply scripts/build-prep-1node-cortx-mgw.sh]: Fixed it.

File: scripts/build-prep-1node-cortx-mgw.sh [hunk 5]
```diff
@@ -71,13 +71,13 @@ echo "========================================="
 
 echo
 echo 'update singlenode.yaml with devices in "path: /dev/loop0" format'
-echo 'Do not use "/dev/$available_device" in cluster yaml'
+echo "Do not use "/dev/$available_device" in cluster yaml"
```
Comment (mehjoshi) [inline scripts/build-prep-1node-cortx-mgw.sh:74]: remove inner pair of double quotes to overcome Codacy warning.

File: scripts/build-prep-1node-cortx-mgw.sh [hunk 6]
```diff
@@ -88,10 +88,11 @@ echo '[client.rgw.8000]'
...
+echo "Current directory: $PWD"
+echo "Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs"
 echo 'check:     hctl status'
 echo 'm0client should be in a started state'
 echo 'To stop the cluster run: ../src/stop.sh'
 echo 'Then:  hctl shutdown'
-echo 'If local directory is used then after reboot of the node,'
-echo 'run: mount "/dev/$available_device" $CORTX_WDIR'
+echo 'If local directory is not used then after reboot of the node,'
+echo "run: mount "/dev/$available_device" $CORTX_WDIR"
```
Comment (mehjoshi) [inline scripts/build-prep-1node-cortx-mgw.sh:98]: remove inner pair of double quotes to overcome Codacy warning.

File: scripts/build-prep-1node-cortx-mgw.sh (hunk 5 above)
Comment (andriytk) [inline scripts/build-prep-1node-cortx-mgw.sh:74]: Quoting is confusing here. Do it like this:
```
echo "Do not use /dev/$available_device in cluster yaml"
```

File: scripts/build-prep-1node-cortx-mgw.sh (hunk 6 above)
Comment (andriytk) [inline scripts/build-prep-1node-cortx-mgw.sh:98]: Fix quoting here please. Same as above.

Comment (andriytk) [inline scripts/build-prep-1node-cortx-mgw.sh:98]: Fix quoting here please:
```
echo "run: mount /dev/$available_device $CORTX_WDIR"
```

**Review by andriytk — APPROVED** [review]

File: scripts/build-prep-1node-cortx-mgw.sh (hunk 5 above)
Comment (madhavemuri) [reply scripts/build-prep-1node-cortx-mgw.sh]: Fixed it.

File: scripts/build-prep-1node-cortx-mgw.sh (hunk 6 above)
Comment (madhavemuri) [reply scripts/build-prep-1node-cortx-mgw.sh]: Fixed it.

**Pull request merged**